ArduCopter/ReleaseNotes.txt

```diff
@@ -1,5 +1,88 @@
+    f) "GPS Glitch" message clarified to "GPS Glitch or Compass error"
+    g) Pre-arm check that configured AHRS is being used (e.g. checks AHRS_EKF_TYPE not changed since boot)
+    h) Pre-arm check that gimbals are healthy (currently only for Gremsy gimbals, others in future release)
+    i) Pre-arm check that all motors are setup
+    j) Pre-arm check that scripts are running
+    k) Pre-arm check that terrain data loaded if RTL_ALT_TYPE set to Terrain
+    l) Pre-arm messages are correctly prefixed with "PreArm:" (instead of "Arm:")
+    m) RC auxiliary switch option for Arm / Emergency Stop
+    n) RC failsafe made pre-arm check (previously only triggered at arming)
+    o) RC failsafe option (see FS_OPTIONS) to continue in Guided obeyed even if GCS failsafe disabled
+    p) TKOFF_RPM_MIN checks all motors spinning before takeoff
+    q) Vibration compensation disabled in manual modes
+7) Scripting enhancements
+    a) CAN2 port bindings to allow scripts to communicate on 2nd CAN bus
+    b) ESC RPM bindings to allow scripts to report engine RPM
+    c) Gimbal bingings to allow scripts to control gimbal
+    d) Pre-arm check bindings (allows scripts to check if pre-arm checks have passed)
+    e) Restarting scripts supported (may require GCS support to be added)
+    f) Semicolon (:) and period (.) supported (e.g both Logger:write() and Logger.write will work)
+    g) VTOL QuickTune script allows easy/safe tuning of quadcopters and multicopters
+8) Sensor driver enhancements
+    a) Benewake H30 radar support
+    b) BMI270 IMU performance improvements
+    c) Logging pause-able with auxiliary switch.  see RCx_OPTION = 165 (Pause Stream Logging)
+    d) Proximity sensor support for up to 3 sensors
+    e) Precision Landing consumes LANDING_TARGET MAVLink message's PositionX,Y,Z fields
+    f) RichenPower generator maintenance-required messages can be suppressed using GEN_OPTIONS param
+    g) TeraRanger Neo rangefinder support
+    h) UBlox GPS support to provide ellipsoid altitude instead of AMSL (see GPS_DRV_OPTIONS)
+9) Bug fixes
+    a) Accel calibration throws away queued commands from GCS (avoids commands being run long after they were sent)
+    b) Cygbot proximity sensor fix to support different orientations (see PRXx_ORIENT)
+    c) Loiter fix to avoid potential wobble on 2nd takeoff (was not clearing non-zero attitude target from previous landing)
+    d) Lutan EFI message flood reduced
+    e) Missions download to GCS corruption avoided by checking serial buffer has space
+    f) Payload place fix so vehicle flies to specified Lat,Lon (if provided).  Previously it could get stuck
+    g) Safety switch disabled if IOMCU is disabled (see BRD_IO_ENABLE=0)
+    h) Takeoff vertical velocity limits enforced correctly even if PILOT_TKOFF_ALT set to a significant height
+10) Developer items
+    a) Custom controller support
+    b) Parameter defaults recorded in @SYS/defaults.parm and onboard logs
```

Comment:
there is no @SYS/defaults, its built into the param ftp protocol

Reply:
ok fixed, txs